src/test/java/gov/healthit/chpl/aqa/stepDefinitions/CollectionsPagesSteps.java

```diff
@@ -47,5 +48,11 @@ public void downloadSEDDetailsFile(final String fileName) throws FileNotFoundExc
         ((JavascriptExecutor) getDriver()).executeScript("arguments[0].click();", link);
         super.checkIfFileIsDownloaded(fileName);
     }
-}
 
+    @Given("^I navigate to \"([^\"]*)\" page$")
+    public void userLoadsCollectionsPage(final String pageName) {
+        getDriver().get(getUrl() + "/#/collections/" + pageName);
```

Comment:
This action does not match the description of the step. A user "navigating" to a page does not do so by entering the URL into their bar, they do it by clicking on the Shortcuts dropdown, then on the name of the page they want to go to. Or by starting on the search page and clicking the big grey box for the collection they want.